samples/nrf9160/http_application_update/README.rst

```diff
@@ -37,8 +39,8 @@ Building and running
 .. include:: /includes/build_and_run.txt
 
 The sample is built as a non-secure firmware image for the nrf9160_pca10090ns board.
-It uses MCUboot and SPM (Secure Partition Manager).
-They are automatically built and merged into the final HEX file when building the sample.
+It automatically includes the :ref:`secure_partition_manager`.
```

Comment:
What about "Because of this, the sample will automatically include the ..." ? 

The way it is stated now makes it sound like the sample selects to include SPM explicitly.

Reply:
OK, updated (here and in build_and_run_nrf9160.txt).